tests/test_routes.py

```diff
@@ -36,7 +36,7 @@ def test_upload_xlsx_successful(flask_test_client, example_pre_ingest_data_file,
     page_html = BeautifulSoup(response.data)
     assert response.status_code == 200
     assert "Return submitted" in str(page_html)
-    assert "We will only contact you using the email you’ve provided, if we need to:" in str(page_html)
+    assert "We’ll do this using the email you’ve provided." in str(page_html)
     send_confirmation_email.assert_called_once()
```

Comment:
Can we also assert that the html contains "Service Desk" and "Arrange a callback"